### PR TITLE
adding-issue-valid-functionality

### LINKED ISSUE #4933: [CI] Label Commenter for new GH issue label: "issue/valid"
### Current Behavior
A new GitHub issue label `issue/valid` has been created for purposes of enhancing issue triage and conveying expectations regarding the procedure of issue assignment.

### Desired Situation
Issues labeled with `issue/valid` will ideally have automated comment added in which the  issue is identified as being verified valid and desirable that action be taken in accordance with the description offered in the issue.

Consideration should be given as to whether a similar comment should be made on issues labeled with `help wanted`. 

### Implementation
Enhance the existing [label-trigger.yaml workflow](https://github.com/layer5io/layer5/blob/be89af60eda10e59683abfcb6a936a46354ea5d8/.github/workflows/label-triggered.yml#L35-L40) to include support for `issue/valid` and potentially `help wanted`.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by PRANJALRANA11**
**Description**

This PR fixes #4933 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7f44154f02d11902afa7dfc421ace4091ea627af at: https://650f074e5abffc3ce7699e79--layer5.netlify.app

**Comment by PRANJALRANA11** [comment]
@leecalcote please review the changes

**Comment by saurabh100ni** [comment]
let's discuss this item on today's website meeting,
please add this agenda in the docs: https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk

**Comment by l5io** [comment]
🚀 Preview for commit f83368c049d598d65b297042a916d91132d202d8 at: https://651440cbd4ade4008fefbb9d--layer5.netlify.app

**Comment by sandramsc** [comment]
Looks good @PRANJALRANA11, workflow includes support for support for _issue/valid_ and _help wanted_.I also used [YAMLlint](https://www.yamllint.com/) to validate the .yml file 👍🏽

**Comment by leecalcote** [comment]
@sandramsc any grammatical corrections that you might suggest or rephrasing is most welcomed.

File: .github/workflows/label-triggered.yml [hunk 1]
```diff
@@ -39,6 +39,30 @@ jobs:
...
+  add-comment-issue-valid:
+    if: github.event.label.name == 'issue/valid'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+              Thanks for opening this issue.It is a verified and valid issue you can take action in accordance
```
Comment (sandramsc) [inline .github/workflows/label-triggered.yml:51]: To increase readability I suggest restructuring this sentence to read as follows:
```suggestion
              Thank you for opening this issue. It has been verified and is considered a valid issue. You can now take action based on the provided issue description. 
```

Comment (sandramsc) [inline .github/workflows/label-triggered.yml:51]: To increase readability I suggest restructuring this sentence to read as follows:
```suggestion
              Thank you for opening this issue. It has been verified and is considered a valid issue. You can now take action based on the provided issue description. 
```

File: .github/workflows/label-triggered.yml [hunk 2]
```diff
@@ -39,6 +39,30 @@ jobs:
...
+  add-comment-help-wanted:
+    if: github.event.label.name == 'help-wanted'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+              Thank you for opening this issue extra attention is needed for this issue.
```
Comment (sandramsc) [inline .github/workflows/label-triggered.yml:63]: To increase readability I suggest restructuring this sentence to read as follows:
```suggestion
             Thank you for opening this issue. It requires special attention, and we appreciate your contribution.
```

**Comment by sandramsc** [comment]
> @sandramsc any grammatical corrections that you might suggest or rephrasing is most welcomed.

Hi Lee, yes, I've added some inline comments with suggestions. 👍🏽

**Comment by PRANJALRANA11** [comment]
@sandramsc commited that

**Comment by PRANJALRANA11** [comment]
@leecalcote will this be good to merge

**Comment by leecalcote** [comment]
> @leecalcote will this be good to merge

Almost. The suggestion verbiage needs refinement.

**Comment by l5io** [comment]
🚀 Preview for commit 4c5c307643e3a1d1fabe0a14272378d7f8cc9b76 at: https://6518edd058a2270c7fa4725f--layer5.netlify.app

**Comment by PRANJALRANA11** [comment]
is this refinement good or do i have to add some more

**Review by leecalcote — CHANGES_REQUESTED** [review]
@PRANJALRANA11, I'll offer a final round of feedback. If the next review requires more changes, let's closeout this PR and allow you to move onto another issue.

1. "You may now proceed with appropriate actions based on the provided issue description"

No, this is inaccurate.

2. The issue doesn't ask for a comment to be added on application of a "help wanted" label. Thanks for making consideration for one, however, this enhancement isn't desired.

**Pull request closed without merging**